packages/design-system-react/src/components/button-link/ButtonLink.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { ButtonLinkSize } from '.';
+import { IconName } from '..';
+import { ButtonLink } from './ButtonLink';
+import README from './README.mdx';
+
+const meta: Meta<typeof ButtonLink> = {
+  title: 'React Components/ButtonLink',
+  component: ButtonLink,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+      description:
+        'Required prop for the content to be rendered within the ButtonLink',
+    },
+    className: {
+      control: 'text',
+      description:
+        'Optional prop for additional CSS classes to be applied to the ButtonLink component',
+    },
+    isDanger: {
+      control: 'boolean',
+      description:
+        'Optional prop that when true, applies error/danger styling to the button',
+    },
+    size: {
+      control: 'select',
+      options: Object.values(ButtonLinkSize),
```

Comment:
Will remove this in a separate PR https://github.com/MetaMask/metamask-design-system/issues/306